linked_list/linked_list.py

```diff
@@ -23,13 +23,44 @@ def __init__(self):
     self.tail = None
 
   def add_to_tail(self, value):
-    pass
+    node = Node(value)
+    if self.head != None:
+      self.tail.set_next(node)
+      self.tail = self.tail.get_next()
+    else:
+      self.head = node
+      self.tail = node
 
   def remove_head(self):
-    pass
+    if self.head != None:
+      headVal = self.head.get_value()
+      if self.head.get_next() != None:
+        self.head = self.head.get_next()
+      else:
+        self.head = None
+        self.tail = None
```

Comment:
These last two if/else statements work; you can probably find a workaround to take out the `else` statement. Also remember that you just want to remove the head, not the tail.